### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by CuriousSugam**
- unnecessary code removed
- unnecessary comments removed 
- used Butterknife to inject views

### CODE REVIEW

File: app/src/main/java/com/lftechnology/activitylogger/WifiActivity.java [hunk 1]
```diff
@@ -34,24 +34,26 @@
...
-    private RecyclerView recyclerView;
-    private LinearLayoutManager layoutManager;
+
 
     private boolean activityBlank = false;
 
     @BindView(R.id.swipeRefreshWifiActivity)
     SwipeRefreshLayout swipeRefreshLayout;
 
+    @BindView(R.id.application_list_wifi_usage)
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/WifiActivity.java:44]: @CuriousSugam change the resourse id name

File: app/src/main/java/com/lftechnology/activitylogger/adapter/NetworkDataAdapter.java [hunk 2]
```diff
@@ -121,21 +123,22 @@ public int getItemCount() {
...
 
-        private ImageView applicationIcon;
-        private TextView applicationName;
-        private TextView receivedBytes;
-        private TextView transmittedBytes;
-        private TextView totalBytes;
-        private ProgressBar progressBar;
+        @BindView(R.id.main_app_icon)
+        ImageView applicationIcon;
+        @BindView(R.id.main_app_name)
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/adapter/NetworkDataAdapter.java:129]: @CuriousSugam add a blank line after each declaration in the case of butterknife

File: app/src/main/java/com/lftechnology/activitylogger/fragments/SecondMainFragment.java [hunk 3]
```diff
@@ -21,25 +21,42 @@
...
+
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.second_main_fragment, container, false);
+        ButterKnife.bind(this, view);
+
         final List<AppDetails> appDetailsList = getArguments().getParcelableArrayList(MainActivity.APP_DETAILS);
-        TextView txtWifi = (TextView) view.findViewById(R.id.tv_menu_wifi);
         txtWifi.setOnClickListener(new View.OnClickListener() {
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/fragments/SecondMainFragment.java]: @CuriousSugam why not use butterknife's onclick here too

File: app/src/main/java/com/lftechnology/activitylogger/viewholders/AllAppsViewHolder.java [hunk 4]
```diff
@@ -7,18 +7,23 @@
 
 import com.lftechnology.activitylogger.R;
 
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
 /**
  * Created by Sugam on 7/27/2016.
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/viewholders/AllAppsViewHolder.java:16]: @CuriousSugam class desc

**Comment by manas-raj-shrestha** [comment]
everything esle seems good. merge this for now

**Comment by CuriousSugam** [comment]
- class description added 
- implemented butterknife for click listeners

**Pull request merged**